content/authors/Carlos-Kahuru/author.md

```diff
@@ -0,0 +1 @@
+Carlos Kaharu is a student at Meru University. Developing  Artificial Intelligence software and applications that a particular business might use is what I'm always interested in. To program systems that will change to suit the business’s needs based on the data collected and analyzed is the main goal.
```

Comment:
Carlos Kaharu is a student at Meru University. Developing  Artificial Intelligence software and applications that a particular business might use is what I'm always interested in. To program systems that will change to suit the business’s needs based on the data collected and analyzed is the main goal.

Reply:
```suggestion
Carlos Kaharu is a student at Meru University. Carlos is interested in developing Artificial Intelligence softwares and applications for businesses to suit their needs. He spends most of his time learning about AI and writing technical articles. 
```